Gems/SimulationInterfaces/Code/Include/SimulationInterfaces/SimulationMangerRequestBus.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <SimulationInterfaces/SimulationInterfacesTypeIds.h>
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/smart_ptr/shared_ptr.h>
+#include <AzFramework/Physics/ShapeConfiguration.h>
+#include <SimulationInterfaces/Result.h>
+
+namespace SimulationInterfaces
+{
+    using SimulationState = uint8_t;
+    class SimulationManagerRequests
+    {
+    public:
+        AZ_RTTI(SimulationManagerRequests, SimulationManagerRequestsTypeId);
+        virtual ~SimulationManagerRequests() = default;
+
+        //! Reload level and removal of all spawned simulation entities.
+        using ReloadLevelCallback = AZStd::function<void(void)>;
+        virtual void ReloadLevel(ReloadLevelCallback completionCallback) = 0;
```

Comment:
I would assume that this will correctly refer code with this documentation.
```suggestion
        using ReloadLevelCallback = AZStd::function<void(void)>;
        //! Reload level and removal of all spawned simulation entities.
        virtual void ReloadLevel(ReloadLevelCallback completionCallback) = 0;
```